tests/kernel/pipe/pipe_api/src/main.c

```diff
@@ -19,13 +19,17 @@ extern void test_pipe_block_put(void);
 extern void test_pipe_block_put_sema(void);
 extern void test_pipe_get_put(void);
 
+/* k objects */
+extern struct k_pipe put_get_pipe;
 /*test case main entry*/
 void test_main(void)
 {
+	k_thread_access_grant(k_current_get(), &put_get_pipe, NULL);
+
 	ztest_test_suite(test_pipe_api,
 			 ztest_unit_test(test_pipe_thread2thread),
```

Comment:
can we do this one as well?

Reply:
I was facing some random issues with it a while back, so stopped using it in user mode. 
Now it seems to work fine.